components/support/rs_client/src/http_client.rs

```diff
@@ -4,11 +4,11 @@
 
 //! This is a simple HTTP client that uses viaduct to retrieve Remote Settings data from the server.
 
+use std::sync::Mutex;
```

Comment:
Aaah should we use `parking_lot::Mutex` here instead of `Mutex`?

At some point in the past we switched to using `parking_lot` because the `Mutex`s there don't get poisoned, I imagine we should probably do the same here

Reply:
Yeah, I think so - they roughly work the same, just with less explicit `unwrap`s!